system/testing/BaseTestCase.cfc

```diff
@@ -311,15 +311,17 @@ component extends="testbox.system.compat.framework.TestCase" accessors="true"{
 	/**
 	* Executes a framework lifecycle by executing an event.  This method returns a request context object that can be used for assertions
 	* @event The event to execute
+    * @uri The uri to execute
 	* @private Call a private event or not
 	* @prePostExempt If true, pre/post handlers will not be fired.
 	* @eventArguments A collection of arguments to passthrough to the calling event handler method
 	* @renderResults If true, then it will try to do the normal rendering procedures and store the rendered content in the RC as cbox_rendered_content
-	* 
+	*
 	* @return coldbox.system.context.RequestContext
 	*/
 	function execute(
-		required event,
+		string event = "",
+        string uri = "",
```

Comment:
Following the lead of the `setNextEvent()` method in ColdBox, would it be more appropriate to have a separate `queryString` argument?  Then you can provide it alongside an event or a route.

Is the `URI` arg meant to just be the route as opposed to the full URL with domain and everything?  The DocBox comment doesn't really specify.  I've never cared for `URI` as it's often used as a synonym for URL, and is at best confusing.  Any thoughts on calling it `route`  since that's basically what I think you're wanting to pass.